documentation/book/proc-changing-a-topic.adoc

```diff
@@ -62,3 +62,9 @@ endif::Kubernetes[]
 ----
 oc replace -f _<your-file>_
 ----
+
+.Additional resources
+* For more information about the API for `KafkaTopics`, see xref:type-KafkaTopic[`KafkaTopic` API documentation]
+* For more information about deploying a Kafka cluster using the Cluster Operator, see xref:cluster-operator-str[]
```

Comment:
I think we can leave the _using the Cluster Operator_ ...or?

Reply:
If you leave the [ ] blank it pulls the title from the section directly (or it should) so I've been doing that because it's less to maintain if we change titles.

Reply:
@ncbaratta for xrefs to what we might be calling the "Schema Reference" I've put in names because the generated heading are really ugly (e.g. "### `KafkaTopic` kind v1alpha1 kafka.strimzi.io" which is simply copied from how these things are documented in k8s, see for example https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.10/#clusterrole-v1-rbac-authorization-k8s-io). Obviously we can easily change the generated ones, but we should bear in mind how things will work in a future where there are multiple versions of the schema. I suppose in that case we would always link to the most up to date version of the schema (and where multiple versions exist they would interlink. Thus I suppose we don't really need the version number in that heading, nor the api group (the `kafka.strimzi.io` bit).

So I'll change those generated section headings to "<type-name> Schema" (or "<type-name> Schema reference"?) if people are happy with the "Schema reference" nomenclature?

Reply:
Oh, I thought @scholzj was asking about the one you had with the empty brackets. I was answering him and not complaining about how you did things :)

Reply:
@ncbaratta @tombentley I'm asking about neither one of those :-). I'm actually asking about this:
_For more information about deploying a Kafka cluster **using the Cluster Operator**, see xref:cluster-operator-str[]_

There is just one way how to deploy Kafka cluster, so it seem unnecessary. No big deal anyway.

Reply:
Ha! I think I've been reading way too many of these comments lately :)